packages/webdriverio/tests/utils/interception/devtools.test.js

```diff
@@ -78,7 +78,7 @@ test('abort request', async () => {
         responseHeaders: [{ name: 'Content-Type', value: 'application/json' }]
     }
     const mock = new NetworkInterception('**/foobar/**')
-    expect(::mock.abort).toThrow()
+    expect(mock.abort.bind(mock)).toThrow()
```

Comment:
I feel like these binds are overkill, especially in a test. Is it not?

Reply:
They are necessary otherwise we would loose the scope and the function would not work.